docs/loader_usage.md

```diff
@@ -0,0 +1,114 @@
+
+- `sdcm.tester.ClusterTester.run_stress_thread`
+- `sdcm.utils.loader_utils.LoaderUtilsMixin.assemble_and_run_all_stress_cmd`
+
+### Stress Command Parameters
+
+Stress commands are specified using these parameters:
+
+- **`stress_cmd`**: Used primarily in longevity tests.
+- **`stress_cmd_<w|r|m>`**: Used in performance tests, with `<w|r|m>` representing write, read, or mixed workloads.
+
+These parameters accept either a single string or a list of strings, each representing a distinct stress command.
+
+### Adjustments to Stress Commands
+
+Stress commands provided by the user undergo the following modifications before execution:
+
+- **Node targeting**: IP addresses of all nodes in the cluster are appended automatically.
+- **Duration adjustment**: Modified according to the test’s `stress_duration` and `prepare_stress_duration` parameters, if present.
+- **HDR Histogram**: if `use_hdrhistogram` is set to `true`, proper arguments are added and SCT handles collection HDR histograms. Supported by `cassandra-stress`, `cql-stress`, `latte` and (possibly) `scylla-bench`; (`ycsb` WIP).
+- **Authentication**: Adjusted based on LDAP parameters (`use_ldap`, `are_ldap_users_on_scylla`) and `authenticator_user` and `authenticator_password`.
+- **Encryption**: Included when `client_encrypt` is enabled, considering additional params like `peer_verification` and `client_encrypt_mtls`.
+- **Multi-DC tests**: Loader targets nodes within the same datacenter.
+- **Rack awareness**: Enabled if `rack_aware_loader` is `true` (stress command targets nodes on the same rack as loader is).
+
+### Execution Modes
+
+Commands are executed in two possible modes based on the `round_robin` setting:
+
+- **Round Robin (`true`)**: Each stress command runs sequentially on loaders, rotating through available loaders.
+- **Parallel Execution (`false`)**: All loaders run the command simultaneously. If `stress_multiplier` (or specific variants like `stress_multiplier_<w|r|m>`) is greater than 1, commands execute multiple parallel instances on each loader. **Avoid this mode** due to data overwrites, which can lead to incorrect data size estimation and increased compaction load.
+
+---
+
+## Preloading Data
+
+Before executing main stress tests, loaders can populate the cluster with initial data using the `prepare_write_cmd` parameter. This is essential for validating read operations. This parameter also accepts single or multiple commands and follows similar execution logic.
+
+---
+
+## Adjusting Load Size
+
+To effectively measure cluster performance, ensure loaders deliver sufficient stress:
+
+- **`n_loaders`**: Specifies the number of loader machines.
+- **`instance_type_loader`**: Defines the loader's hardware specification (use CPU-optimized instances for better results). Search test configuration files for examples (depends on cloud provider: `instance_type_loader` for AWS, `azure_instance_type_loader` for Azure, `gce_instance_type_loader` for GCE)
+
```

Comment:
Need to add statement that there is bunch of options for the K8S case and it is not covered here.

There are following options:
- k8s_n_loader_pods_per_cluster
- k8s_loader_run_type

Reply:
you can add this section later as a follow-up

Reply:
addressed